lib/opensusebasetest.pm

```diff
@@ -1058,7 +1058,7 @@ sub wait_boot_past_bootloader {
     # Starting with GNOME 40, upon login, the activities screen is open (assuming the
     # user will want to start something. For openQA, we simply press 'esc' to close
     # it again and really end up on the desktop
-    send_key('esc') if match_has_tag('gnome-activities');
+    send_key('esc') if match_has_tag('gnome-activities') || match_has_tag('opensuse-welcome-gnome40-activities');
```

Comment:
Why not use `gnome-activities` as tag in the `opensuse-welcome-gnome40-activities` needles?

Tags shouldn't be distro and version specific anyway.

Reply:
Good question.
The code has been updated.